config/system_factory_presets.go

```diff
@@ -77,6 +77,7 @@ func defaultProductionConfig() mutableNodeConfig {
 
 	cfg.SetDuration(GOSSIP_CONNECTION_KEEP_ALIVE_INTERVAL, 1*time.Second)
 	cfg.SetDuration(GOSSIP_NETWORK_TIMEOUT, 30*time.Second)
+	cfg.SetDuration(GOSSIP_RECONNECT_INTERVAL, 5*time.Second)
```

Comment:
Why so long? I think 0.5 second should be fine here too.

Reply:
reducing to 1 second... don't want it too low in case several nodes are permanently offline or unreachable. then, reconnect attempt rate is multiplied by the number of offline nodes and it might become noisy.